worker/uniter/modes.go

```diff
@@ -344,21 +382,27 @@ func ModeConflicted(curl *charm.URL) Mode {
 	return func(u *Uniter) (next Mode, err error) {
 		defer modeContext("ModeConflicted", &err)()
 		// TODO(mue) Add helpful data here too in later CL.
-		if err = u.unit.SetUnitStatus(params.StatusBlocked, "upgrade failed", nil); err != nil {
+		// The spec says we should set the workload status to Error, but that's crazy talk.
```

Comment:
does the spec really deal with modeconflicted here?  I don't believe that anyone who wrote the spec has the faintest understanding of what modeconflicted actually implies...